core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2981,6 +2981,13 @@ public String toString() {
               + "an exponential backoff. This property determines the maximum duration to retry for"
               + " before giving up. Note that, this value is set to 5s for fs and fsadmin CLIs.")
           .build();
+  public static final PropertyKey USER_RPC_MAX_INBOUND_MESSAGE_SIZE =
```

Comment:
You can reuse `USER_NETWORK_MAX_INBOUND_MESSAGE_SIZE` which is currently used for worker clients.

Reply:
Unless you see any problem bumping the default to 100MB?

Reply:
I am fine with that. The data path messages are larger than control path most of the time.

Reply:
OK. RPC messages are different from stream messages. However we should be fine sharing a same value as it defines the max only.